### DISCUSSION TITLE
GetAvailablePurchases response when subscribing with requestSubscription

### DISCUSSION THREAD

**Discussion by oajh04** [hyochan/react-native-iap#2229] [Q&A]
hello guys!
we are currently implementing regular subscription payments with requestSubscription, and even though we succeeded in finishTransaction, the subscription payment history that we paid before when calling getAvailablePurchases is being responded, and I wonder if it works normally.

**Comment by andresesfm** [comment]
What "subscription payment history"? where are you seeing this?
